src/seedu/addressbook/data/person/ReadOnlyPerson.java

```diff
@@ -82,4 +82,5 @@ default String getAsTextHidePrivate() {
         }
         return builder.toString();
     }
+    
```

Comment:
Be careful of unnecessary or accidental changes here as it can cause unnecessary merge conflict.